lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlMulti.pm

```diff
@@ -97,7 +98,8 @@ XML
 
     my $i=0;    
     for($i=0;$i<@bams;$i++){
-        my $path = abs_path($bams[$i]);
+        my $uri = URI->new($bams[$i]);
+        my $path = $uri->scheme ? $bams[$i] : abs_path($bams[$i]);
```

Comment:
`has_recognized_scheme` might be more correct than `scheme` but I don't think it actually matters because of the constructor that is used.

`file:///some/path` is a valid URI.  Would you want to `abs_path` the path in that case or leave it alone?